### DISCUSSION TITLE
Speedtest = 200Mbps  Unifi Router 1.4 Gbps  Same Speedtest Server

### DISCUSSION THREAD

**Discussion by craigccfl** [alexjustesen/speedtest-tracker#1324] [Q&A]
Appreciate all the Devs' work on this project. I do have a question, Unifi DMP built in speedtest states 1.4 Gbps & Speedtest-Tracker states 211 Mbps using the same speed test server/location. I've had other speedtest trackers do the same in the past.  Everything is hardwired to a 1 Gig switch with Cat 6 and the server running my dockers has a 2.5 G network card. Can anyone explain the discrepancy ?

Thanks in advance,

**Comment by alexjustesen** [comment]
I'm pretty sure UI has their own speedtest servers, at least based on this 2019 blog post from them: https://blog.ui.com/2019/08/13/ubiquiti-launches-a-speed-test-network/.

I'd make sure nothing else on your network is taking up bandwidth when testing and also try some different servers.

**Reply by craigccfl** [reply]
Correct, Unify is using Comcast and Speedtest-tester  is using Ookla, both located in the same state & city, 20-miles from my location. After doing some GTS your good suggestions are inline with other postings found on Unifi forums. I'll try moving the speedtest-tracker docker to another lxc sever as a stand alone. I'm still exploring choke points in my network. It might just be time to buy a gateway with more horsepower, my home network has a lot of cameras along with some home lab servers that keep growing. 😎  I'll post an update if I find something that closes the gap between direct router and speedtest-tracker results.

**Reply by craigccfl** [reply]
"I'll try moving the speedtest-tracker docker to another lxc sever as a stand alone. Only running 15% load on the network, external & internal.

Update: No Joy  Had the same results with other tracking dockers. It's not specific to this docker.  Thanks for the help all.
